cmd/task/task.go

```diff
@@ -135,6 +136,7 @@ func run() error {
 	pflag.DurationVarP(&flags.interval, "interval", "I", 0, "Interval to watch for changes.")
 	pflag.BoolVarP(&flags.global, "global", "g", false, "Runs global Taskfile, from $HOME/{T,t}askfile.{yml,yaml}.")
 	pflag.BoolVar(&flags.experiments, "experiments", false, "Lists all the available experiments and whether or not they are enabled.")
+	pflag.IntVar(&flags.maxRuns, "max-runs", task.MaximumTaskCall, "Set the maximum number of runs per task before being considered infinte or cyclic and therefore terminated.")
```

Comment:
No need to change this now. Just opening this up for thoughts and opinions as naming is always hard. Some other possibilities: 

- `--limit` - Nice and short, but might be too generic (could clash with something like `include-limit` if we wanted it)
- `--call-limit`
- `--max-task-calls` - Very explicit, but longer. I don't think length is a huge issue with these flags as they're most likely to be used in CI or scripts.

Reply:
Also, I'd like to shorten the description. I couldn't find more concise (yet complete) description.